cluster-scope/overlays/ocp-prod/externalsecrets/metallb-memberlist-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  name: memberlist
+  namespace: metallb-system
+spec:
+  backendType: secretsManager
+  data:
+    - key: cluster/ocp-prod/metallb/memberlist
+      name: secretkey
```

Comment:
See my comments on #35 regarding the structure of secrets in AWS.

Reply:
I fixed this too.